homeassistant/components/reolink/media_source.py

```diff
@@ -285,14 +289,9 @@ async def _async_generate_resolution_select(
         """Allow the user to select the high or low playback resolution, (low loads faster)."""
         host = self.data[config_entry_id].host
 
-        # re-evaluating supporting ext playback, NVR does not support it and it is extra work for cameras
-        # disabling for now but leaving code in place for future consideration
-        hasExt = (
-            False  # host.api.api_version("live", channel) == 1 and not host.api.is_nvr
```

Comment:
This will not be accepted in a HA PR as this is basically commented out code.
Besides the media_source requires 100% test coverage, now you have created a piece of code that is unacessible and can therefore not be covered by tests, that means you cannot get 100% coverage and can not get the tests to pass.

You will have to remove the code if you don't want to use it.

Reply:
I commented out the code so I could test, and note why I did it, most likely it will be removed as I dont think supporting ext is really necessary for the downloads. I only implemented it because I tend to be a completionist.